shared/components/Conference/index.js

```diff
@@ -1,28 +1,5 @@
 import React from 'react';
-import SiteFooter from '../SiteFooter';
-import Hero from '../Hero';
-import RedBadgerBanner from '../RedBadgerBanner';
-import MailingList from '../MailingList';
-import ConferenceAbout from '../ConferenceAbout';
-import NextConferenceEvent from '../NextConferenceEvent';
-import ShareYourIdeas from '../ShareYourIdeas';
-
-const first = {
```

Comment:
This will probably clash with https://github.com/redbadger/react.london/pull/167